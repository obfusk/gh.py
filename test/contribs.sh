#!/bin/bash
"$PY" gh.py contribs obfusk \
  | jq '[.[]|select(.name|contains("bun"))]'
