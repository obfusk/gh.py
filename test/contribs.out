[
  {
    "name": "rubygems/bundler",
    "count": 1,
    "contrib": true
  }
]
